docs/src/pages/guides/batfish-helpers.md

```diff
@@ -252,6 +253,69 @@ class PageShell extends React.Component {
 {{JSON.stringify(splitPages, null,2)}}
 ```
 
+## Prepare sitemap
+
+`prepareSitemap` creates a data file that lists all files with `hideFromSearchEngines: true` or `splitPage: true` set in the page's frontMatter. Use this generated file with Batfish's [`sitemap.ignoreFile`](https://github.com/mapbox/batfish/blob/main/docs/configuration.md#sitemap) option to generate a sitemap that excludes any paths in the prepared file.
```

Comment:
Do you only need to worry about `prepareSitemap` if you're planning on hiding some files from search engines? Or will all dr-ui / Batfish sites need the `sitemapIgnore: ... prepareSitemap()` combo?

Reply:
You only need to introduce this Batfish helper if you have pages that you want to exclude, otherwise the sitemap will build without intervention.